blueman/plugins/applet/RecentConns.py

```diff
@@ -150,15 +145,13 @@ def on_manager_state_changed(self, state):
             self.item.set_sensitive(True)
             adapters = self.parent.Manager.get_adapters()
 
-            self.Adapters = {}
-            for adapter in adapters:
-                self.Adapters[str(adapter.get_object_path())] = str(adapter["Address"])
+            self._adapters = {str(adapter.get_object_path()): str(adapter["Address"])
+                              for adapter in adapters}
 
-            for i in reversed(self.items):
-                try:
-                    i["device"] = self.get_device_path(i)
-                except (AdapterNotFound, DeviceNotFound):
-                    pass
+            for i in self.items:
```

Comment:
You dropped ``reversed()`` here but left it in ``on_device_removed`` is that intentional? Also ``self.initialize()`` already sorts so I am confused why the reversed was there in the first place?

Reply:
The reverse iteration is necessary there as the loop removes the current item from the list. It does not have any use here. It is a pre ac5eb6d8a3c09b116a07258d1a8914f667e197bc leftover where this loop actually used to remove as well.